### PR TITLE
Fix manual typos

### PR DESCRIPTION
**Pull request by m-holger**

### CODE REVIEW

File: manual/encryption.rst [hunk 1]
```diff
@@ -47,24 +47,24 @@ Password Protection
   Password protection is distinct from encryption. This point is often
   misunderstood. A PDF file can be encrypted without being
   password-protected. The intent of PDF encryption was that there
-  would be two passwords: a *user password* and an *owner password*.
+  would be two passwords: an *user password* and an *owner password*.
```
Comment (jberkenbilt) [inline manual/encryption.rst:50]: `a` is correct here, at least in American usage. It's phonetic, not lexical, and we use `a` before the `y` sound in `yoozer`.

**Review by jberkenbilt — APPROVED** [review]
I accept the suggestions that I agree with. There may be some idiomatic differences between American and British usage. Mostly I think my use of "the user and owner password" in a general sense is fine, and while "user and owner passwords" also works, I don't think the wording change is necessary, and since my natural way of saying it is "the user and owner password", I'll leave it the way it is. Thanks again for all your review.

**Comment by jberkenbilt** [comment]
Selected changes incorporated locally.

File: manual/encryption.rst (hunk 1 above)
Comment (m-holger) [reply manual/encryption.rst:50]: Same in British English. It's just that some people can't tell their consonants from their elbows.

**Comment by jberkenbilt** [comment]
I've heard people say things like "an historic occasion" even while pronouncing the "h" sound so I wasn't sure if the "y" sound, which is also just barely an elbow, er, consonant is similar. My guess is that people used to drop the h sound and day something that sounded like an istoric. This is how language changes. (I fancy myself something of an amateur linguist... I've read too many books.)

**Comment by m-holger** [comment]
I believe history is actually the French  histoire and therefore the "h" would have traditionally been silent - for Cockneys it still is.
Similar to herb (French herbe), where I believe some people even today don't pronounce the "h".

**Comment by jberkenbilt** [comment]
Yup, sounds right. There are still remnants, like "hour", "honor", etc. I just think it's funny that people correctly say, "an hour" and "an honor" but then go on to say "an historic". I've never heard anyone say, "I had an happy day today." It seems to only be "historic". At least that's the only word I can think of like that. Then there's the "n" shifting around, but that's a whole nother thing. (Do people say that in your preferred dialect?)

**Pull request closed without merging**